Task1/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1
+{
+    class Program
+    {
+        public static void Main()
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Мцыри М.Ю. лермонтов");
+            Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine("Однажды русский генерал Из гор к Тифлису проезжал");
```

Comment:
А лучше в две строки, стихи всё-таки. И с авторской пунктуацией.
```suggestion
            Console.WriteLine("Однажды русский генерал\nИз гор к Тифлису проезжал;");
```